src/pages/detail/Detail.js

```diff
@@ -1,7 +1,374 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import Modal from "react-modal";
+import ModalOne from "./ModalOne";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
 
 const Detail = () => {
-  return <div>Detail</div>;
+  // const [isModal, setIsModal] = useState(false);
+  const [modalIsOpen, setIsOpen] = React.useState(false);
```

Comment:
react 패키지에서 useState를 import했기 때문에 React 객체에 접근하지 않아도 됩니다.
```suggestion
  const [modalIsOpen, setIsOpen] = useState(false);
```

Reply:
네! 수정완료했습니다